proto/lavanet/lava/rewards/params.proto

```diff
@@ -40,4 +40,10 @@ message Params {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
     (gogoproto.nullable)   = false
   ];
+
+  google.protobuf.Duration ibc_iprpc_expiration = 7 [
```

Comment:
when adding a new param you must set it in a migrator or we will get a panic

Reply:
done. see https://github.com/lavanet/lava/commit/12b0582b29f896f95c15c770300717651fddb92c

migrator was tested both locally and with state sync